all_emoji_names.txt

```diff
@@ -215,7 +215,6 @@ batman
 battery
 bazel
 beach_with_umbrella
-beans
```

Comment:
Were these leaving intentional?

Reply:
I believe the names were changed upstream as of #369, but it seems like some of the changes were missed?

Reply:
weird - the lines removed in this PR were **added** in #369.

I only have a loose understanding of what `all_emoji_names.txt` is for, but why would they be added and then removed?